exporters/trace/zipkin/zipkin.go

```diff
@@ -35,58 +38,105 @@ type Exporter struct {
 	serviceName string
 	client      *http.Client
 	logger      *log.Logger
+	o           options
 }
 
 var (
 	_ export.SpanBatcher = &Exporter{}
 )
 
 // Options contains configuration for the exporter.
-type Options struct {
+type options struct {
```

Comment:
Keeping this unexported sounds good. In https://github.com/open-telemetry/opentelemetry-go/issues/536 it seems like the consensus was to call this `Config`, but it can be for a separate PR (addressing https://github.com/open-telemetry/opentelemetry-go/issues/536 directly) if we don't want to address here.